app/domain/errors.rb

```diff
@@ -449,6 +449,11 @@ module Constraints
         code: "CONJ00069E"
       )
 
+      EmptyAnnotationsListConfigured =  ::Util::TrackableErrorClass.new(
```

Comment:
We call it `Constraints`, not `Annotations`:
```suggestion
      RoleMissingAnyConstraints =  ::Util::TrackableErrorClass.new(
```

Reply:
Done

Reply:
Done